tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -44,17 +44,14 @@ trait Sharing {
 	 *
 	 * @var SimpleXMLElement[]
 	 */
-	private array $lastShareDataByUser = [];
+	private ?SimpleXMLElement $userSharesData = null;
 
 	/**
-	 * Contains the share id of the last share that was created by each user,
-	 * either using the Sharing API or on the web UI.
+	 * Contains all the user share data
 	 *
-	 * @var string[]
+	 * @var array
 	 */
-	private array $lastShareIdByUser = [];
-
-	private ?string $userWhoCreatedLastShare = null;
+	private array $createdUserShares = [];
```

Comment:
So, will there be another store for group-share and link-share as well?

Reply:
There's a separate store for link-share. Group shares and user shares are stored in this store.

Reply:
So, `createdUserShares` can be misleading. Maybe use `createdUserGroupShares`